tests/Unit/NumberTest.php

```diff
@@ -242,3 +242,23 @@
     '-1 is not positive or zero' => [-1, false],
     '-10000 is not positive or zero' => [-10000, false],
 ]);
+
+it('can check if equal to', function (int|string|float $number, int|string|float $comparison) {
+    expect(Number::of($number)->isEqualTo(Number::of($comparison)))->toBe(true);
+})->with([
+    '1 is 1' => [1, 1],
+    '10.01 is 1001' => [1001, 1001],
+    '0 is 0' => [0, 0],
+    '-101.10 is -10110' => [-101.10, -101.10],
+]);
+
+it('can get Number in cents', function (int|string|float $number, int $result) {
+    expect(Number::of($number)->inCents())->toBe($result);
+})->with([
+    '1.00 as string is 1' => ['1.00', 100],
+    '10.01 is 1001' => [10.01, 1001],
+    '0 is 0' => [0.00, 0],
+    '101.10 as string is 10110' => ['101.10', 10110],
```

Comment:
Probably test with a float as well?

And perhaps do the same for negative values?